public/documentation/_data.json

```diff
@@ -1,50 +1,108 @@
 {
   "about": {
-    "title": "About"
```

Comment:
Now I'm going to jump to the other argument for _data.json...  Is there a reason to have just a single _data.json instead of _data_en.json, _data_zh.json, ...  :)